beets/ui/__init__.py

```diff
@@ -1122,9 +1122,12 @@ def _configure(options):
     # special handling lets specified plugins get loaded before we
     # finish parsing the command line.
     if getattr(options, 'config', None) is not None:
+        overlay_path = True
         config_path = options.config
         del options.config
         config.set_file(config_path)
+    else:
+        overlay_path = False
```

Comment:
Alternatively, we could just set one variable: change the name of `config_path` to `overlay_path` and have it set either to a string or to None.